pubspec.yaml

```
@@ -16,7 +16,8 @@ dependencies:
   flutter_riverpod: 2.6.1
   freezed_annotation: ^3.0.0
   hooks_riverpod: ^2.6.1
-  intl: 0.19.0 # must be 0.19.0 for flutter_localizations
+  http: ^1.3.0
+  intl: 0.19.0 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)